### ISSUE TITLE
Consenting Glorious Eggroll for providing Proton-GE and GE-Proton

### ISSUE THREAD

**Issue by TheEvilSkeleton** [bottlesdevs/Bottles#1582]
This was reported by @RaduAvramescu. It seems like Glorious Eggroll, the maintainer of Proton-GE, is quite against third parties using Proton-GE outside of Steam, and recommends to use wine-ge-custom. Or so that's the impression I get.

In the [README](https://github.com/GloriousEggroll/proton-ge-custom#1-running-non-steam-games-with-proton-outside-of-steam-is-not-supported-do-not-ask-for-help-with-this):

> (1) RUNNING NON-STEAM GAMES WITH PROTON OUTSIDE OF STEAM IS NOT SUPPORTED. DO NOT ASK FOR HELP WITH THIS:

I suggest to either drop later versions of Proton-GE altogether, or the preferable approach to contact Glorious Eggroll. If we take the latter approach, I suggest to openly ask them so we can easily link the proof, in case anyone wants to use it against us.

**Comment by mirkobrombin** [comment]
I use the issue to explain that Bottles uses Proton "only" for Proton prefixes created through Steam, only using the experimental function that can be activated from Bottles preferences.

Steam Proton prefixes are not bottles created via Bottles. Once the experiment is activated, Bottles will start listing the prefixes created by Steam among the classic bottles. These are managed differently, in fact it is not possible to change the version of the components: runner (Wine), DXVK, VKD3D .. in fact Bottles finds and uses the version of proton used by Steam for that specific prefix. All actions taken in Botles for this type of prefix go through the Steam runtime (soldier for the most recent). The purpose of this feature is only and exclusively to tweak the prefixes by changing for example the registry keys or enable mangohud, gamemode etc.

GE-Proton (as explained by the author) should only be used in Steam. In the past it has simply been provided like all other runners as well it worked and we have not received any reports to the contrary. However, we do not want to suggest this as users may report problems to GE that it will not (rightly) be ready to fix. One of the plans for GE-Proton was to allow the user to change Proton version for a Steam prefix directly from Bottles, this is still up for debate and may never see the light.

Over time we have added Wine-GE among Bottles runners, these are technically the builds the user should use for classic bottles.

Currently it is allowed to select Proton as runner for "normal" bottles as Bottles treats it as a normal component classified as "wine". There has been some initial work to differentiate wine from proton in Bottles but this has never been completed and is what I would like to do in the near future.

I have been planning (for months now) to contact GE to find out if this is correct and possible, unfortunately due to the many commitments, development and my work I could not do.

In summary: Proton in Bottles should ONLY be used via the experimental "Steam Proton prefixes support" feature.

**Comment by ghtesting2020** [comment]
I don't remember which games now as this was months ago but there were times where games simply wouldn't run if I did not use the proton runner (outside of steam). Maybe it was bugs in the Vaniglia version at the time I do not know but I remember having to switch to ProtonGE quite a few times. Also performance seems better in ProtonGE versus wineGE despite it not being made to be used outside of Steam prefixes. I've never actually had a problem running games outside of Steam with Bottles ProtonGE runner. I switched to the WineGE when it was added and have not given Caffe a shot so I cannot comment about compatibility anymore.

**Comment by mirkobrombin** [comment]
I test everything with Caffe (software, dependencies, installers), all is built with Caffe in mind. Also Caffe is a custom TKG build which follow Wine releases (7.10 now) and add some Proton patches; Proton/GE-Proton/Wine-GE follow Valve's Wine releases (7.0). Wine releases versions very often by fixing bugs and improving compatibility and performance.
